{"students": [{"student_id": "nullpointer", "source": "N/A", "email": "npem", "student_number": "5555555555", "section": "N/A", "last": "npl", "first": "npf"}], "assignment": "E2", "results": {"E2.e2tester.GraphTest": {"passes": {"testReflectionMethodGetNumEdges": "the getNumEdges() method", "testReflectionMethodGetNeighbours": "the getNeighbours(Node<T> node) method", "testAddEdgeID2NotInGraph": "the addEdge(int id1, int id2) method", "testConstructer": "the Graph constructor", "testReflectionMethodAddNode": "the addNode(int id, T value) method", "testReflectionMethodGetNodes": "the getNodes() method", "testGetNodeNotInGraph": "the getNode(int id) method", "testAddEdgeIDNoNodesInGraph": "the addEdge(int id1, int id2) method", "testGraphMultipleNodesByNode": "a Graph with multiple Nodes", "testAreAdjacentID2NotInGraph": "the areAdjacent(int id1, int id2) method", "testReflectionMethodAreAdjacentNode": "the areAdjacent(Node<T> node1, Node<T> node2) method", "testReflectionMethodGetNumNodes": "the getNumNodes() method", "testGraphMultipleNodesByID": "a Graph with several Nodes", "testReflectionMethodToString": "the toString() method", "testReflectionMethodAddEdgeNode": "the addEdge(Node<T> node1, Node<T> node2) method", "testAreAdjacentID1NotInGraph": "the areAdjacent(int id1, int id2) method", "testReflectionMethodAreAdjacentId": "the areAdjacent(int id1, int id2) method", "testReflectionMethodGetNode": "the getNode(int id) method", "testAreAdjacentIDNoNodesInGraph": "the areAdjacent(int id1, int id2) method", "testGetNodesOnNewGraph": "a Graph with no Nodes", "testReflectionMethodAddEdgeId": "the addEdge(int id1, int id2) method", "testAddEdgeID1NotInGraph": "the addEdge(int id1, int id2) method"}, "failures": {"testAddEdgeNode2NotInGraph": {"description": "the addEdge(Node<T> node1, Node<T> node2) method", "message": "java.lang.NullPointerException", "details": "java.lang.NullPointerException\n\tat e2soln.Graph.addEdge(Graph.java:143)\n\tat E2.e2tester.GraphTest.testAddEdgeNode2NotInGraph(GraphTest.java:582)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)\n\tat sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.lang.reflect.Method.invoke(Method.java:606)\n\tat org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)\n\tat org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)\n\tat org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)\n\tat org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)\n\tat org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)\n\tat org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)\n\tat java.util.concurrent.FutureTask.run(FutureTask.java:262)\n\tat java.lang.Thread.run(Thread.java:745)\n"}, "testAddEdgeNode1NotInGraph": {"description": "the addEdge(Node<T> node1, Node<T> node2) method", "message": "java.lang.NullPointerException", "details": "java.lang.NullPointerException\n\tat e2soln.Graph.areAdjacent(Graph.java:96)\n\tat e2soln.Graph.addEdge(Graph.java:141)\n\tat E2.e2tester.GraphTest.testAddEdgeNode1NotInGraph(GraphTest.java:563)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)\n\tat sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.lang.reflect.Method.invoke(Method.java:606)\n\tat org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)\n\tat org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)\n\tat org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)\n\tat org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)\n\tat org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)\n\tat org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)\n\tat java.util.concurrent.FutureTask.run(FutureTask.java:262)\n\tat java.lang.Thread.run(Thread.java:745)\n"}, "testAddEdgeNodeNoNodesInGraph": {"description": "the addEdge(Node<T> node1, Node<T> node2) method", "message": "java.lang.NullPointerException", "details": "java.lang.NullPointerException\n\tat e2soln.Graph.areAdjacent(Graph.java:96)\n\tat e2soln.Graph.addEdge(Graph.java:141)\n\tat E2.e2tester.GraphTest.testAddEdgeNodeNoNodesInGraph(GraphTest.java:598)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)\n\tat sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.lang.reflect.Method.invoke(Method.java:606)\n\tat org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)\n\tat org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)\n\tat org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)\n\tat org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)\n\tat org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)\n\tat org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)\n\tat java.util.concurrent.FutureTask.run(FutureTask.java:262)\n\tat java.lang.Thread.run(Thread.java:745)\n"}, "testAreAdjacentNode1NotInGraph": {"description": "the areAdjacent(Node<T> node1, Node<T> node2) method", "message": "java.lang.NullPointerException", "details": "java.lang.NullPointerException\n\tat e2soln.Graph.areAdjacent(Graph.java:96)\n\tat E2.e2tester.GraphTest.testAreAdjacentNode1NotInGraph(GraphTest.java:512)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)\n\tat sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.lang.reflect.Method.invoke(Method.java:606)\n\tat org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)\n\tat org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)\n\tat org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)\n\tat org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)\n\tat org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)\n\tat org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)\n\tat java.util.concurrent.FutureTask.run(FutureTask.java:262)\n\tat java.lang.Thread.run(Thread.java:745)\n"}, "testAreAdjacentNode2NotInGraph": {"description": "the areAdjacent(Node<T> node1, Node<T> node2) method", "message": "areAdjacent(node1, node2) should throw a NoSuchNodeException if node2 is not in the Graph", "details": "java.lang.AssertionError: areAdjacent(node1, node2) should throw a NoSuchNodeException if node2 is not in the Graph\n\tat org.junit.Assert.fail(Assert.java:88)\n\tat E2.e2tester.GraphTest.testAreAdjacentNode2NotInGraph(GraphTest.java:532)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)\n\tat sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.lang.reflect.Method.invoke(Method.java:606)\n\tat org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)\n\tat org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)\n\tat org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)\n\tat org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)\n\tat org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)\n\tat org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)\n\tat java.util.concurrent.FutureTask.run(FutureTask.java:262)\n\tat java.lang.Thread.run(Thread.java:745)\n"}, "testGraphOneNode": {"description": "a Graph with a single Node", "message": "no self edges are allowed", "details": "java.lang.AssertionError: no self edges are allowed\n\tat org.junit.Assert.fail(Assert.java:88)\n\tat org.junit.Assert.assertTrue(Assert.java:41)\n\tat E2.e2tester.GraphTest.testGraphOneNode(GraphTest.java:212)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)\n\tat sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.lang.reflect.Method.invoke(Method.java:606)\n\tat org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)\n\tat org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)\n\tat org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)\n\tat org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)\n\tat org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)\n\tat org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)\n\tat java.util.concurrent.FutureTask.run(FutureTask.java:262)\n\tat java.lang.Thread.run(Thread.java:745)\n"}, "testAreAdjacentNodeNoNodesInGraph": {"description": "the areAdjacent(Node<T> node1, Node<T> node2) method", "message": "java.lang.NullPointerException", "details": "java.lang.NullPointerException\n\tat e2soln.Graph.areAdjacent(Graph.java:96)\n\tat E2.e2tester.GraphTest.testAreAdjacentNodeNoNodesInGraph(GraphTest.java:547)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)\n\tat sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.lang.reflect.Method.invoke(Method.java:606)\n\tat org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)\n\tat org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)\n\tat org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)\n\tat org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)\n\tat org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)\n\tat org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)\n\tat java.util.concurrent.FutureTask.run(FutureTask.java:262)\n\tat java.lang.Thread.run(Thread.java:745)\n"}}, "errors": {}}, "E2.e2tester.NoSuchNodeExceptionTest": {"passes": {"testIsCheckedException": "that NoSuchNodeException is a checked exception", "testConstructorNoArgs": "the NoSuchNodeException constructor with no arguments", "testConstructorOneArg": "the NoSuchNodeException constructor with one argument"}, "failures": {}, "errors": {}}}, "origin": "/home/anya/projects/at/u-of-t-autotester/uam/testjam/submissions/nullpointer/E2", "date": "2015-07-21T12:34:14"}